promotion/src/main/resources/application.yml

```
@@ -55,4 +65,6 @@ management:
       probability: 1.0
   zipkin:
     tracing:
-      endpoint: http://localhost:9411/api/v2/spans
\ No newline at end of file
+      endpoint: http://localhost:9411/api/v2/spans
+
+schedule-key: schedule
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)